src/main/java/net/openhft/chronicle/wire/WireMarshaller.java

```diff
@@ -817,6 +817,8 @@ public static Object create(@NotNull Field field, @Nullable Object defaultObject
                     case "net.openhft.chronicle.bytes.Bytes":
                         return new BytesFieldAccess(field);
                     default:
+                        if (Jvm.isJava14Plus() && (boolean) Jvm.getMethod(Class.class, "isRecord").invoke(type))
```

Comment:
Can you be cached
static {
Method isRecord = null;
try {
   if (Jvm.isJava14Plus())
       isRecord = Jvm.getMethod(Class.class, "isRecord");
} catch (Exception expected) {
   isRecord = null; 
}
IS_RECORD = isRecord;
}
// later
if (IS_RECORD != null && IS_RECORD.invoke(type))

Reply:
Done